internal/loki/logql/parser.go

```diff
@@ -45,7 +44,7 @@ func ParseMatchers(input string) ([]*labels.Matcher, error) {
 	}
 	matcherExpr, ok := expr.(*matchersExpr)
 	if !ok {
-		return nil, logqlmodel.ErrParseMatchers
+		return nil, errors.New("only label matchers are supported")
```

Comment:
Could there be a function somewhere which checks for `ErrParseMatchers`? E.g. if it sees this error it'd ignore it?

Reply:
No, this function is not even used in our codebase. I keept it here because we could have a usecase for it to remove a loki import but want to test that out in another pr